ci/kubo-deployment.yml

```diff
@@ -123,12 +123,30 @@ resources:
     url: {{slack-url}}
 
 jobs:
+- name: run-unit-tests
+  plan:
+  - get: git-kubo-deployment
+  - get: git-kubo-release
+    version: every
+    trigger: true
```

Comment:
I don't understand what the result of having trigger=true on kubo-release but not on kubo-deployment. Does this mean that if kubo-deployment changes but kubo-release doesn't, the run-unit-tests step will be skipped?